portfolio/src/main/java/com/google/sps/servlets/FactsServlet.java

```diff
@@ -0,0 +1,48 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import java.util.ArrayList;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import com.google.gson.Gson;
+import com.google.sps.data.FactsData;
```

Comment:
Nit: Imports typically are put in alphabetical order, so these `com.google` imports would go above `import java.io.IOException`. Don't feel obligated to submit a fix for this, I was just pointing it out for learning purposes :)

Reply:
Ok, got it! I would keep a note on this in the future!